client/src/components/App.jsx

```diff
@@ -1,8 +1,34 @@
 // import react, axios, recommendations list
 import React from 'react';
 import Axios from 'axios';
+import styled from 'styled-components';
 import List from './RecommendationsList.jsx';
 
+const Time = styled.p`
+  text-align: center;
+  color: green;
+`;
+
+const Wrapper = styled.div`
+  background-color: green;
+`;
+
+// make Recommendations styled div to encompass entire module
```

Comment:
Still need these comments? Or would it help to clean them up?

Reply:
I'll clean those up in the next push.